### DISCUSSION TITLE
BRAINSTORMING: Mission, Vision, and Values of the DAO

### DISCUSSION THREAD

**Discussion by ricardroyo** [iota-community/Community-Governance#28] [General]
I'm opening up this thread to discuss the mission, vision and values of the DAO. 
The goal is to build a shared understanding of what the DAO is and to answer some basic questions to provide direction and purpose to the DAO.

UPDATE: I'm going to create a bunch of comments with some general questions for everyone to answer. It is sort of a brainstorming exercise that maybe can help us define a bit more what the community wants. Feel free to add further questions

**Comment by ghost** [comment]
i'm worried about the whale plutocracy problem

1,000 people could vote for {X}

but if one whale (or a few whales colluding) have enough holdings, their 1 or 2 or 3 votes {Y} could outweigh the 1,000

and it's a hard problem, no easy solution

and i'm not concerned with whether or not a whale is malicious or altruistic in their intent

the point is "those with the money make the rules" is not the way it should be, whale plutocracy should not be the future

**Reply by ricardroyo** [reply]
I share your fear Ben and I doubt we will be able to avoid it completely. We do not live in a perfect world and technology is still not there. But that should not stop us from using the tools we have at our disposal and try to use them with the best intentions, even if we fail.

Let's try to keep this thread on focus and move this to another thread though. I would love to know more about what other people would want for the DAO to be.

**Comment by ricardroyo** [comment]
Why should the DAO exist?

**Reply by Phyloiota** [reply]
Because it can be a more free and unregulated way of supporting project and persons than the current EDF System allows. Due to the EDF beeing part of IF and this is a foundation under german law it has some strict requirements that seem sometimes to hinder granting funds to projects. A DAO would only need to follow the rules created by the community and written into the code and will therefore be free from any laws or regulations that countries set.

**Reply by Deep-Sea-888** [reply]
1. The DAO should exist to use the 65 Ti that is left unclaimed to support the IOTA protocol and ecosystem in a positive manner.
2. The DAO should exist to allow the IOTA community to show that a Decentralized Autonomous Organization can be formed from the ground up and run in a fully decentralized and "feeless" manner. This DAO can be a test case and more so a positive example in which "other communities" can then go forth and build their own DAO on the Tangle. Given the Tangle's fee'less nature, ISCP's (IOTA Smart Contract Protocol), Decentralized Identifiers, and Tokenization allow for the foundation, user, and economically friendly protocol structure to form a DAO. The community showing that we can build a DAO can be an ignition for other entities and organizations to form their own DAO's on the Tangle.
3. There are many limitations the IOTA Foundation has in place due to being regulated within Germany as a non-profit organization. For example, the IF is limited on how they fund marketing campaigns and are not allowed to pay to be listed on certain exchanges. Just for example sake, the DAO would allow for project proposals to conduct a marketing campaign or pay to be listed on a specific exchange. Would these be voted for, who knows, but the fact that there is the option to submit this type of proposal certainly expands the ability to grow the IOTA eco-system greater than the current abilities of the IF.
4. As a hedge to risk! There is without no doubt a risk to the current development of the IOTA protocol. What is that risk? It is for any reason unknown that the IOTA Foundation must shut down. It doesn't matter what that reason is, it's just the fact that it is a possibility. Regardless if that possibility is extremely small, it is still a risk. So what is that consequence? All funding to developers, to partner project managers and representatives, to all projects being funded by the EDF, etc. All of this "could" come to a complete halt. Having the DAO and the 65 Ti is simply a hedge to offset this extremely small risk. With the DAO in place in the extreme circumstance this happened, the DAO could go on funding the development and expansion of the IOTA protocol.
5. My last statement as to why the DAO should exist is because it is an amazing opportunity to spend the time, and the funds, to build the foundations of a DAO. Looking into the future there is no doubt that one day the IOTA Foundation will not be a significant organization developing the protocol, developing the eco-system, and growing the eco-system. To be a truly decentralized system not only must the protocol act autonomously without the IF, but some foundation of voting and approval should be in place on how the protocol is edited, upgraded, etc. This initial DAO is a great chance to start building these foundational blocks and an entity that one day may take over the management of the protocol and the Tangle.

**Comment by ricardroyo** [comment]
Who should the DAO serve?

**Reply by Phyloiota** [reply]
This is pure my opinion: The DAO should serve everyone and every project that obeys to the core principles that i see as goals of IOTA. Serving in this case would not mean pay out funds, but more serve that this principles will be contained and preserved and strengthened
- Transactions on the Tangle are always free 
- Everyone can run a node and be a validator
- Everyone can send messages to the tangle
- no permissions, no censorship, no control, full decentralization of the Tangle
- All decisions in the Tangle are done by the protocol as implemented in the code

**Reply by Deep-Sea-888** [reply]
What is, "to serve"? The DAO's objective? The DAO's objective should be to support any projects that support the IOTA ecosystem in a positive manner. For that aspect the DAO should serve IOTA's community. As below, the community is anyone that supports or builds on the IOTA protocol and does it in a positive manner.

**Comment by ricardroyo** [comment]
Who IS the IOTA Community?

**Reply by Phyloiota** [reply]
It is so difficult... everyone that wants IOTA in its feeless and decentralized form to be a successful protocol that is used by people all over the world?

**Reply by Deep-Sea-888** [reply]
I agree anyone that who supports the IOTA protocol in a positive manner can be considered part of the community.

**Comment by ricardroyo** [comment]
What are the values that the DAO should stand for?

**Reply by Phyloiota** [reply]
A feeless, decentralized Tangle, open and unrestricted for everyone.
Serving the public good, serving environmental friendly development
Everything else needs a lot of discussions to reach a consensus...

**Reply by Deep-Sea-888** [reply]
The DAO's values should be - 
- Distributed and community-run
- Transparent in all aspects
- The DAO is in service and support of the IOTA community
- An attempt to seek the fairest and democratic voting system
- To promote and support projects that do no direct harm (to people, animals, and the environment)
---------->> (This is a tough one. Let's say a project wants to create a DEX on the Tangle? One could easily say that a person could lose their life savings trading on the DEX. Is that harmful? Is that moral and ethical? Personally, I think a DEX is great but can not be responsible for a person's irresponsible disregard of financial management.
- Funded projects are for the direct support and expansion of the IOTA Protocol in a positive manner
-------->> This can be anything from technical development for the Tangle to, development of Dapps for the IOTA ecosystem, marketing campaigns which the IF can not do due to being a non-profit, to fund conferences, meetups, or fundraisers, and even potentially simply fund a social program (ie. to fund poor African communities to get programming courses, or to teach 3rd world country programmers how to code on the Tangle, or even purchase things like Ras Pi's etc.)

**Comment by Phyloiota** [comment]
Who should be allowed to vote?

**Reply by Deep-Sea-888** [reply]
I would like to say anyone within the community, but to give tangible attachment I think it should be community members that hold IOTA's, and, Node owners. If hosting a node can be identified through DiD possibility, I think node owners should have the rights to vote regardless of holding IOTA's. Additionally, I think IOTA holders should be a basis to vote.

**Comment by Phyloiota** [comment]
Brainstorming: How can someone apply for funding?

**Reply by ebk1960** [reply]
Depending on the type of application, e.g, someone wants to do a video on setting up a Node with RPi4, or build a universal battery pack using IOTA wallet. Each of these have different funding demands plus or more management. 

All submissions should be voted on, but some would need to capture more votes if they would require more financial resources/DAO Executive Board input.

**Reply by Deep-Sea-888** [reply]
All applications should follow a designated template. Personally, I like the IOTA EDF structure in their .pdf's. 

So every proposal would have to include things such as:
- Introduction
- Project proposal
- How does your project support the IOTA Ecosystem?
- Execution Summary
- Project/Technical Breakdown (Include a timeline to follow)
- Project Deliverables broken down by Milestones and funds required for each deliverable

As structure within the DAO I think it would be good to have spam controls in place.
        Spam Controls:
- Project submittals must be submitted with an associated amount of locked funds.
- --- For projects requesting $0to $10,000 (50 Mi locked staking)
- --- For projects requesting $10,000 to $100,000 (100 Mi locked staking)
- --- For Projects requesting $100,001 to $1,000,000 (1 Gi locked staking)
- --- For Projects requesting >$1,000,000 (10 Gi locked staking)
note: Values are just examples. Locked staking may be simply donating to the DAO (which would be lower amounts). The idea is to have some spam control in place to prevent a wave of never-ending submittals. The increase of donation/staking should deter ridiculous high fund requests. If someone is asking for over a million dollars, they clearly should have no problem gathering X amount of Giota's. If they can't get X amount of Giota's then they probably can't manage a million-dollar project.

**Comment by Phyloiota** [comment]
Brainstorming: Should there be a need to deposit funds as a security to propose something?

**Reply by ebk1960** [reply]
I think for higher valued/riskier development projects the proposer should put up their own funds or capture delegated funds to support their project idea.

**Reply by Deep-Sea-888** [reply]
I posted above with the "how to propose". I certainly think some type of fund deposit or donation should be in place to prevent spamming. If a fund locking can be technically possible this would probably be the best method along with an increasing amount equal to the funds being requested. If technically staking/locking is too difficult, we can simply put set amounts which get donated to the DAO fund. Essentially people would have to pay a certain amount to submit a proposal.

**Comment by Phyloiota** [comment]
Brainstorming: Should there be a lock period for funds that used to cast a vote for or against a proposal?

**Reply by ebk1960** [reply]
I don't think funds should be locked up enable a vote. This requirement could reduce voting activity of the DAO community.

**Reply by Deep-Sea-888** [reply]
I do like the idea that individuals with smaller funds can lock up funds to get a high voting percentage. I believe other DAO's have similar abilities. This by all means should not take away from people being able to vote without locking up funds as well. One thing about IOTA I always thought was amazing was their tech always offers flexibility and options. Hopefully, the DAO can follow a similar model.

**Comment by Phyloiota** [comment]
Brainstorming? Should certain projects not be allowed for funding grants?

**Reply by ebk1960** [reply]
Yes, certain activities should be outside scope for financial support. This should be embedded in the Articles of Association/DAO constitution.
